Makefile

```diff
@@ -358,6 +358,7 @@ manifests.crds: controller-gen manifests.versions ## Generate CustomResourceDefi
 manifests.versions: kustomize yq
 	$(YQ) eval '.appVersion = "$(VERSION)"' -i charts/kong-operator/Chart.yaml
 	$(YQ) eval '.image.tag = "$(VERSION)"' -i charts/kong-operator/values.yaml
+	$(YQ) eval '.tag = "$(VERSION)"' -i charts/kong-operator/values.yaml
```

Comment:
This is not really desired, right? I believe we shouldn't add this but we should remove https://github.com/Kong/gateway-operator/blob/8b56f338712cb6229ad1df89f5442fd41ac03ee8/charts/kong-operator/values.yaml#L99

```suggestion
```

Reply:
Why not? String with version in file `charts/kong-operator/values.yaml` is in lines `3` and `99` (`.tag`) so both should be set to the same value by automation. Or am I missing something?

Reply:
> Or you mean to remove this field entirely, because it's redundant?

Sorry for not making this more clear. Yes, `.tag` is redundant. Thanks for taking care of this.